libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -667,7 +667,19 @@ type ConnectionAPI =
                :> Get '[Servant.JSON] (SearchResult Contact)
            )
 
-type BrigAPI = UserAPI :<|> SelfAPI :<|> ClientAPI :<|> PrekeyAPI :<|> UserClientAPI :<|> ConnectionAPI
+type SecondFactorAuthAPI =
+  Named
+    "send-verification-code"
+    ( Summary "Send a verification code to a user's email address"
+        :> CanThrow SecondFactorAuthDisabled
```

Comment:
There is also `400 no-such-verifiable-action` (or something to that end), if something other than `generate-scim-token` is sent in the body.

Reply:
I like it but, since the request body is typed, Servant will respond with bad request in that case. I don't think it adds much value to have a custom error here, does it?

Reply:
Not sure, I think it is not. But I also do not see it explicitly added for any of the other endpoints.